internal/mq/mqimpl/rocksmq/client/consumer.go

```diff
@@ -72,4 +72,7 @@ type Consumer interface {
 
 	// GetLatestMsgID get the latest msgID
 	GetLatestMsgID() (int64, error)
+
+	// check preCreated topic whether vaild or not
+	CheckPreTopicValid(topic string) error
```

Comment:
Prefer to name `IsEmpty()`

Reply:
it checks two things.1. is topic existed? 2. is topic empty? So I think CheckPreTopicValid is more suitable.

Reply:
CheckTopicValid is better